README.md

```diff
@@ -143,10 +146,12 @@ and max number of retries
     * [.updateEventMetadataForProvider(consumerOrgId, projectId, workspaceId, providerId, eventCode, body)](#EventsCoreAPI+updateEventMetadataForProvider) ⇒ <code>Promise.&lt;object&gt;</code>
     * [.deleteEventMetadata(consumerOrgId, projectId, workspaceId, providerId, eventCode)](#EventsCoreAPI+deleteEventMetadata) ⇒ <code>Promise.&lt;object&gt;</code>
     * [.deleteAllEventMetadata(consumerOrgId, projectId, workspaceId, providerId)](#EventsCoreAPI+deleteAllEventMetadata) ⇒ <code>Promise.&lt;object&gt;</code>
-    * [.createWebhookRegistration(consumerOrgId, integrationId, body)](#EventsCoreAPI+createWebhookRegistration) ⇒ <code>Promise.&lt;object&gt;</code>
-    * [.getWebhookRegistration(consumerOrgId, integrationId, registrationId)](#EventsCoreAPI+getWebhookRegistration) ⇒ <code>Promise.&lt;object&gt;</code>
-    * [.getAllWebhookRegistrations(consumerOrgId, integrationId)](#EventsCoreAPI+getAllWebhookRegistrations) ⇒ <code>Promise.&lt;object&gt;</code>
-    * [.deleteWebhookRegistration(consumerOrgId, integrationId, registrationId)](#EventsCoreAPI+deleteWebhookRegistration) ⇒ <code>Promise.&lt;object&gt;</code>
```

Comment:
I am wondering since these are breaking changes, shouldn't we mark these as deprecated first and only then remove them completely.

Reply:
if they want to use the older methods, they can use the previous version. WDYT? @sandeep-paliwal is that the right way to go?

Reply:
I disagree. 
A version update should not be reason enough for clients to update their integrations.
Even with the digital signature verification feature, we [first added support](https://github.com/adobe/aio-lib-events/pull/19) (on 20th Jan), and after the complete deprecation of the old signature verification, we [removed the HMAC signature verification from SDK](https://github.com/adobe/aio-lib-events/pull/29) (on 16th July).
So clients atleast had a 6 month window for updating their integrations.

Reply:
I would favor removing the v1 and updating the version number accordingly and putting a warning in the release note